mfr/providers/osf/provider.py

```diff
@@ -66,23 +66,24 @@ def __init__(self, request, url):
             metadata_url = download_url.replace('/file?', '/data?', 1)
             metadata_request = await self._make_request('GET', metadata_url)
```

Comment:
I see what you mean. I could see it either way. It returns a request object, but it also represents the response to an HTTP request.

Reply:
`._make_request()` calls `aiohttp.request()` which returns a response object: [link](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L739). I am ok calling it request. If we were to change it, it would be everywhere I think.

Reply:
Change made.